src/read/elf/relocation.rs

```diff
@@ -736,3 +757,165 @@ impl<Endian: endian::Endian> Relr for elf::Relr64<Endian> {
         }
     }
 }
+
+/// Compact relocation
+///
+/// The specification has been submited here: <https://groups.google.com/g/generic-abi/c/ppkaxtLb0P0/m/awgqZ_1CBAAJ>.
+#[derive(Debug, Clone, Copy)]
+pub struct Crel {
+    /// Relocation offset.
+    pub r_offset: u64,
+    /// Relocation symbolindex.
+    pub r_sym: i64,
+    /// Relocation type.
+    pub r_type: i64,
```

Comment:
While the linked spec says `r_sym` and `r_type` are signed, LLVM used `uint32_t`. (The deltas are SLEB128 though.) I plan to change this.